modules/sharedIdSystem.js

```diff
@@ -11,7 +11,8 @@ import { coppaDataHandler } from '../src/adapterManager.js';
 import {getStorageManager} from '../src/storageManager.js';
 
 const GVLID = 887;
-export const storage = getStorageManager({gvlid: GVLID, moduleName: 'pubCommonId'});
+const MODULE_TYPE = 'fpid-module';
```

Comment:
with this fix in place, let's go ahead and remove line 13, correct? Prebid was only a vendor on the deprecated 3rd party component of this.